turbo.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "https://turbo.build/schema.json",
-  "pipeline": {
+  "globalPassThroughEnv": ["NODE_ENV", "CI"],
```

Comment:
ENV variables are not passed automatically to subtasks anymore, vitest relied on `CI` not to run in `WATCH` mode.

I added `NODE_ENV` as some dependencies might need it.